src/main/java/org/apache/sling/serviceusermapping/impl/ServiceUserMapperImpl.java

```diff
@@ -85,7 +87,13 @@
                      "default mapping is applied which uses the service user \"serviceuser--\" + bundleId + [\"--\" + subServiceName]")
         boolean user_enable_default_mapping() default true;
 
+        @AttributeDefinition(name = "Require Validation",
+                description = "Do validators have to be present and validate the service users?")
```

Comment:
To be consistent with the suggested description of "Required Validators" this might be

> If true, a service user is only valid if all the validators listed in the 'required validators' accept it

Reply:
yup